dig_test.go

```diff
@@ -709,6 +733,34 @@ func TestInvokeFailures(t *testing.T) {
 		require.Contains(t, err.Error(), "dig.type2 isn't in the container")
 	})
 
+	t.Run("unmet constructor dependency", func(t *testing.T) {
```

Comment:
Yeah, as in a dependency/requirement that wasn't met. Same wording is used by the previous test.